### DISCUSSION TITLE
Should there be a control.limits minimum speed?

### DISCUSSION THREAD

**Discussion by pqhf5kd** [pybricks/support#1358] [Ideas]
My setup:
- Lego train
- City Hub
- Boost external motor
- PyBricks V3.3.0

When using motor.run below 150 deg/s, PID control is not able to maintain a steady speed in my use case. In combination with a slow control.limits acceleration value, the jerky movement is obvious for a few seconds.

I would around this when I'm not using a control.limits acceleration value by skipping motor.run speeds between -150 and 150.

Should there be a minimum speed value under control.limits?

**Comment by laurensvalk** [comment]
150 deg/s or approximately "15%" speed is about the lowest these motors can do. They have a lot of internal friction so anything slower tends to lead to stick slip behavior, especially with larger attachments.

We've considered limiting the speeds or automatically setting them to a minimum speed but found that this was more confusing than helpful to most users. I don't know if this is the right answer though.

Or maybe we should round anything below 75 to 0 and above to 150?
